src/components/12-modal/modal.njk

```diff
@@ -0,0 +1,93 @@
+<button class="rvt-button" data-modal-trigger="{{ id }}">{{ buttonText }}</button>
+
+<div class="rvt-modal"
+  id="{{ id }}"
+  role="dialog"
+  aria-labelledby="{{ modalTitleId }}"
+  aria-hidden="true"
+  {% if ariaDescribedby %}aria-describedby="{{ ariaDescribedby }}"{% endif %}
```

Comment:
For ones such as this, you might consider using `{% for attribute in attributes %}` logic instead and just adding them to the yaml file as
```
attributes:
    - label: "ariaDescribedby"
       value: "whateverValueYouWant"
    - label: "data-modal-dialog"
       value: "true"
```

Reply:
The `aria-describedby` and `data-modal-dialog` attributes are now moved into a for loop.